chat/chat-tool/tool/pom.xml

```diff
@@ -98,7 +98,13 @@
     <dependency>
       <groupId>taglibs</groupId>
       <artifactId>standard</artifactId>
-      <version>1.0.4</version>
+      <version>1.1.2</version>
+      <scope>runtime</scope>
+    </dependency>
+    <dependency>
+        <groupId>javax.servlet</groupId>
+        <artifactId>jstl</artifactId>
+	<scope>runtime</scope>
```

Comment:
the scope looks wrong here i would remove it...

Reply:
It looks like the jstl jar isn't needed after all. (That's what I started with when troubleshooting 12.x to compare with trunk where trunk had a jstl jar deployed.) The runtime scope for the taglibs jar doesn't look needed either. I'll revise my commit.